python/src/robyn/visualization/transformation_visualization.py

```diff
@@ -0,0 +1,43 @@
+# pyre-strict
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import pandas as pd
+import numpy as np
+from typing import List, Tuple
+
+
+class TransformationVisualizer:
+    @staticmethod
+    def plot_adstock(plot: bool = True) -> Optional[Tuple[plt.Figure, plt.Figure]]:
```

Comment:
plot method, taking plot=true/false doesn't make sense. Is this creating a plot plt.Figure vs actually printing it?